src/modal/_pera-wallet-modal.scss

```diff
@@ -53,39 +56,115 @@
 }
 
 .pera-wallet-connect-modal__close-button {
-  width: 30px;
-  height: 30px;
+  width: 40px;
+  height: 40px;
 
   margin: 0;
   padding: 0;
 
-  box-shadow: 1px 2px 2px rgba(0, 0, 0, 0.08);
-  background-color: white;
+  background: #333333;
+  border: 1.5px solid rgba(255, 255, 255, 0.08);
+  box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.15);
 
   border-radius: 100%;
 }
 
 .pera-wallet-connect-modal__body {
   position: relative;
+  top: 50%;
+  left: 50%;
 
-  width: 440px;
+  width: 655px;
   max-width: calc(100vw - 80px);
 
-  padding: 24px;
+  padding: 28px;
+
+  background-color: #edeffb;
 
-  background-color: white;
   box-shadow: 1px 2px 2px rgba(0, 0, 0, 0.08);
 
-  border-radius: 4px;
+  border-radius: 24px;
+
+  transform: translate(-50%, -50%);
+
+  &::before {
```

Comment:
Let's fix the z-index problem here maybe we can directly use it as a background for `pera-wallet-connect-modal__body` instead of pseudo selector

<img width="671" alt="image" src="https://user-images.githubusercontent.com/5108459/173813486-3e9343ec-f414-43cd-97ab-d1778eaff343.png">

Reply:
I updated background z-index. I think we need before because we need to change background opacity.